install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,107 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
```

Comment:
... Trigger, meaning ... (consider adding a comma?)

Reply:
+1 to adding the comma